build/nightly-build/job-template.yml

```diff
@@ -0,0 +1,116 @@
+parameters:
+  name: ''
```

Comment:
This file is very similar to the `job-template.yml` that we already have for CI builds. Would it make sense to instead use that template, and then define variables and steps based on a parameter passed down to the template? I.e: add a parameter called... isNightlyBuild: false (defaults to false)... then when you reference the template from the nightly build you pass that parameter down as `true`. Then in the template when `isNightlyBuild == true` you run the build and restore steps differently. That way you only maintain one base template file.

Reply:
Yes, I saw what was done for codeCoverage, for nightly build, the logic difference is more as you can see there are 5 steps that is nightly build only, and there are several steps that nightly build don't need. So I use separate template here to keep the logic simple but I don't have strong preference here, let's discuss offline.

---
In reply to: [343777088](https://github.com/dotnet/machinelearning/pull/4444#discussion_r343777088) [](ancestors = 343777088)

Reply:
Offline discussed with Santiago, will merge these template file into

---
In reply to: [343828680](https://github.com/dotnet/machinelearning/pull/4444#discussion_r343828680) [](ancestors = 343828680,343777088)